src/Uno.UI/Microsoft/UI/Xaml/Controls/TreeView/TreeViewNode.Events.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	public partial class TreeViewNode
+	{
+		internal event TypedEventHandler<TreeViewNode, IVectorChangedEventArgs> ChildrenChanged;
+
+		//TODO:
+		//internal event TypedEventHandler<TreeViewNode, DependencyPropertyChangedEventArgs> ExpandedChanged;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)